### PR TITLE
Translate -zh file into Chinese

### PR DESCRIPTION
**Pull request by jennyliang220**
Translated from the [origin English file](https://github.com/w3c/w3c-website-templates-bundle/blob/main/translations/w3c_website_templates_bundle%2Bintl-icu.en.yaml)

This is the third and last file to translate, the rest 2 files are [here](https://github.com/w3c/w3c-website-frontend/pull/61)

### CODE REVIEW

**Comment by jennyliang220** [comment]
Please don't merge until all the `TODO`s are removed.

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 1]
```diff
@@ -1,201 +1,254 @@
...
-    intro: __error.intro {status_code} {status_text}
-    request_id: __error.request_id {request_id}
-    contact: __error.contact {params}
-    faq: __error.faq
+    title: 错误{status_code} - {status_text}
+    intro: >-
+        <p>服务器返回"{status_code} - {status_text}"错误。</p>
+    request_id: >-
+        <p>请求ID: {request_id}</p>
+    contact: __error.contact {params}   #TODO: not in .en file
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:206]: Hi, @jean-gui . `error.contact` and `error.faq` don't exist in the -en file. Should I remove them?

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 2]
```diff
@@ -1,201 +1,254 @@
...
-        __page.default_description
-    skip_to_content: __page.skip_to_content
+        万维网联盟 (W3C) 是一个国际组织。众多会员组织、全职员工及广大群众通力合作，制定并完善网络标准。 （TODO）
+    skip_to_content: 转到内容
     translations: >-
-        __page.translations
-    translationsarialabel: __page.translations.arialabel
-    toc: __page.toc
-    related_content: __page.related
+        切换语言： (TODO)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:10]: @xueyuanjia , 请关注所有带有**TODO**的翻译内容。

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 1 above)
Comment (jean-gui) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:206]: Yes, they can very likely be removed.

Comment (jennyliang220) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml]: removed

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 3]
```diff
@@ -1,201 +1,254 @@
...
     lang:
         dir: ltr
-        options: __page.lang.options
-    default_title: __page.default_title
+        options: 语言
+    default_title: 万维网联盟 (W3C)
     default_description: >-
-        __page.default_description
-    skip_to_content: __page.skip_to_content
+        万维网联盟 (W3C) 是一个国际组织。众多会员组织、全职员工及广大群众通力合作，制定并完善网络标准。 （TODO）
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:7]: 万维网联盟 (W3C) 是一个国际标准组织。众多会员组织、全职员工及广大社区通力合作，制定并完善网络标准。

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 4]
```diff
@@ -1,201 +1,254 @@
...
+        sponsor: 赞助
         sponsorurl: /zh-hans/sponsor/
-        donate: _footer.links.donate
+        donate: 捐赠
         donateurl: /zh-hans/donate/
-        privacy_policy: __footer.links.privacy_policy
+        privacy_policy: 隐私条款
         privacy_policyurl: /zh-hans/policies/privacy/
-        legal: __footer.links.legal
+        legal: 法务 (TODO)
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:36]: 隐私或隐私条款？

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 5]
```diff
@@ -1,201 +1,254 @@
...
     blog:
         filters:
-            search: __listing.blog.filters.search
-            search_button: __listing.blog.filters.search_button
-            all: __listing.blog.filters.all
-            categories: __listing.blog.filters.categories
-            categories_label: __listing.blog.filters.categories_label
-            archives: __listing.blog.filters.archives
-            archives_label: __listing.blog.filters.archives_label
+            search: 搜索文章 (TODO post翻译成文章、博文、帖子?)
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:77]: 搜索文章

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 6]
```diff
@@ -1,201 +1,254 @@
...
+            search_button: 搜索
+            all: 全部
+            categories: 浏览类目
+            categories_label: 文章类目
+            archives: 查看归档
+            archives_label: 文章归档 (TODO archives是归档 还是分类?)
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:83]: 文章归档

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 7]
```diff
@@ -1,201 +1,254 @@
...
+            types: 重大事件类型
+            categories: 类目
+            search_button: 搜索
+            clear: 清除所有
     newsletters:
         filters:
-            archives: __listing.newsletters.filters.archives
-            archives_label: __listing.newsletters.filters.archives_label
+            archives: 查看归档
+            archives_label: 新闻搞归档 (TODO：跟press_releases区别？)
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:106]: typo"搞"，应为新闻稿（news），press release 一般译为“官方新闻通稿”或“媒体新闻稿”。

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 8]
```diff
@@ -1,201 +1,254 @@
...
+        members_include: "{count}成员组织参与{ecosystem_name}生态系统, 包括:"
+        view_all: 查看所有在{ecosystem_name}中的成员 (TODO)
+        get_involved: "如何协作:"
+        join: 加入我们
+        joinurl: /membership/join
+        donate: 捐赠
         donateurl: /donate/
     content_slider:
-        title: __components.content_slider.title
+        title: 组织机构如何看待我们 (TODO)
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:136]: 不是很理解“组织机构如何看待我们”的译文，英文原文或页面链接是什么呢？

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 9]
```diff
@@ -1,201 +1,254 @@
...
-            pressrelease: __components.crosslinks.types.pressrelease
-            pressreleases: __components.crosslinks.types.pressreleases
-            page: __components.crosslinks.types.page
+            blogpost: 博客
+            ecosystem: 生态系统
+            ecosystems: 生态系统
+            event: 事件
+            events: 事件
+            news: 新闻
+            pressrelease: 新闻稿
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:145]: pressrelease: 新闻稿-->“官方新闻通稿”或“媒体新闻稿”。

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 10]
```diff
@@ -1,201 +1,254 @@
...
-        credit_title: __post.meta.credit_title
+            由:
+        credit_title: 作者及出版日期
         published: >
-            __post.meta.published
-        skiplink: __post.meta.skiplink
-    notes_heading: __post.notes_heading
+            出版:
+        skiplink: 跳至第{count}条评论
+    notes_heading: 注释说明 (TODO: Notes在这里啥意思？)
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:156]: 有特定上下文吗？通常notes翻译成“注释说明”可以；如果专指W3C技术文档的Note应译为“备忘”或“小组备忘”。

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 11]
```diff
@@ -1,201 +1,254 @@
...
+        heading: 相关内容
+        heading_tags: 标签
+        heading_ecosystems: 生态系统
+        heading_groups: 组
+        heading_specifications: 技术标准
     feed:
-        title: __post.feed.title
-        link: __post.feed.link {feed_type}
+        title: 相关的 RSS
+        link: 订阅我们的{feed_type}推荐页 (TODO: feed翻译成 推荐页/信息源/聚合页?)
```
Comment (xueyuanjia) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:165]: +1 to 信息源，或“动态消息”。

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 4 above)
Comment (jennyliang220) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:36]: privacy_policy 是隐私条款，legal 的上下文是「policies」，见：
`legalurl: /zh-hans/policies/`

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 8 above)
Comment (jennyliang220) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:136]: `What organizations think about us`, 见 https://github.com/w3c/w3c-website-templates-bundle/blob/main/translations/w3c_website_templates_bundle%2Bintl-icu.en.yaml#L138

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 10 above)
Comment (jennyliang220) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:156]: https://github.com/w3c/w3c-website-templates-bundle/blob/main/translations/w3c_website_templates_bundle%2Bintl-icu.en.yaml#L158

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 4 above)
Comment (xueyuanjia) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:36]: ok, so how about "政策法规"?

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 12]
```diff
@@ -1,205 +1,256 @@
...
+    skip_to_content: 转到内容
     translations: >-
-        __page.translations
-    translationsarialabel: __page.translations.arialabel
-    toc: __page.toc
-    related_content: __page.related
+        切换语言：
+    translationsarialabel: 翻译
+    toc: 页面内容跳转至链接
+    related_content: 在这个部分 (TODO)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:17]: Hi @jean-gui, I see this config is used in [related.html.twig](https://github.com/w3c/w3c-website-templates-bundle/blob/851d5158757555bb59018e038594c09ac54f8dcc/templates/components/styles/related.html.twig#L3), but can you give a screenshot of how it's used in the page?

Is it a recommendation title, or starting words of a paragraph?

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 13]
```diff
@@ -1,205 +1,256 @@
...
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        Copyright © {date, date, ::y} W3C <sup>®</sup> (<a href="https://www.csail.mit.edu/" hreflang="en"><abbr
+        title="麻省理工学院">MIT</abbr></a>, <a
+        href="https://www.ercim.eu/" hreflang="en"><abbr
+        title="欧洲信息学和数学研究联合会">ERCIM</abbr></a>,
+        <a href="https://www.keio.ac.jp/">庆应义塾大学 Keio</a>, <a href="https://ev.buaa.edu.cn/">北京航空航天大学 Beihang</a>) <a
+        href="/Consortium/Legal/ipr-notice">使用章程适用 (TODO)</a>.
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:25]:
```suggestion
        href="/Consortium/Legal/ipr-notice">使用章程适用</a>.
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 14]
```diff
@@ -1,205 +1,256 @@
...
         legalurl: /zh-hans/policies/
-        cookies: __footer.links.cookies
+        cookies: Cookie
         cookiesurl: /zh-hans/cookies/
-        system_status: __footer.links.system_status
+        system_status: 系统状态
         system_statusurl: https://status.w3.org/
-        twitter: __footer.links.twitter
-        github: __footer.links.github
+        twitter: W3C 在推特 (TODO)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:46]:
```suggestion
        twitter: W3C 在推特
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 15]
```diff
@@ -1,205 +1,256 @@
...
+        github: W3C 在 GitHub (TODO)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:47]:
```suggestion
        github: W3C 在 GitHub
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 16]
```diff
@@ -1,205 +1,256 @@
...
 banner:
     beta:
         text: >-
-            __banner.beta.text
-        icon: __banner.beta.icon
+            这是一个全新的网站，<a href="mailto:team-website-redesign@w3.org">请给予反馈</a>帮助我们建设完善网站。
+        icon: 测试版图标 (TODO)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:52]:
```suggestion
        icon: 测试版图标
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 17]
```diff
@@ -1,205 +1,256 @@
...
-    home: __breadcrumbs.home
+    home: 主页
 pagination:
-    first: __pagination.first
-    previous: __pagination.previous
-    next: __pagination.next
-    last: __pagination.last
-    page: "__pagination.page {page}"
+    first: 首页
+    previous: 上一<span class="visuallyhidden">页</span>  (TODO visuallyhidden?)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:62]: Hi @jean-gui, could you explain how is `visuallyhidden` used? In what situation can we see it, or not see it?

image

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 18]
```diff
@@ -1,205 +1,256 @@
...
     people:
-        tel: "__components.people.tel__Tel: <a href=\"tel:{number}\">{number}}</a>"
-        email: "__components.people.email__Email: <a href=\"mailto:{email}\">{email}</a>"
+        tel: "电话: <a href=\"tel:{number}\">{number}</a>"
+        email: "邮箱: <a href=\"mailto:{email}\">{email}</a>"
     evangelists:
-        list_title: __components.evangelists.list_title__Our {ecosystem_name} Evangelists
+        list_title: >-
+            Our {ecosystem_name} { count, plural,
+                =0 { 布道者 } (TODO: evangelists)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:126]:
```suggestion
                =0 { 布道者 }
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 19]
```diff
@@ -1,205 +1,256 @@
...
-        title: __components.members.title
-        members_include: "__components.members.members_include {count} member organizations are involved in the web payment ecosystem, including:"
-        view_all: __components.members.view_all
-        get_involved: __components.members.get_involved
-        join: __components.members.join
-        joinurl: /membership/join/
-        donate: __components.members.donate
+        title: W3C 会员
+        members_include: "{count}成员组织参与{ecosystem_name}生态系统, 包括:"
+        view_all: 查看所有在{ecosystem_name}中的成员 (TODO)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:133]:
```suggestion
        view_all: 查看所有在{ecosystem_name}中的成员
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 10 above)
Comment (jennyliang220) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:160]: @jean-gui , this section is in `visuallyhidden`, can you give a screenshot of the context?
https://github.com/w3c/w3c-website-templates-bundle/blob/ac5a4463773292d94f208d881592cebd8fcfbea2/templates/pages/basic.html.twig#L43-L46

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 20]
```diff
@@ -1,205 +1,256 @@
...
 notes:
     info:
-        default_title: __notes.info.default_title
+        default_title: 信息
     successes:
-        default_title: __notes.successes.default_title
+        default_title: 成功
     warnings:
-        default_title: __notes.warnings.default_title
+        default_title: 警示 (TODO: Warning)
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:184]:
```suggestion
        default_title: 警示
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 21]
```diff
@@ -1,205 +1,256 @@
...
     errors:
-        default_title: __notes.errors.default_title
+        default_title: 出现了一些问题  #TODO: There is a problem
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:186]:
```suggestion
        default_title: 出现了一些问题
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 22]
```diff
@@ -1,205 +1,256 @@
...
-    speakers: __events.speakers
-    host: __events.host
-    category: __events.category
-    website: __events.website
+    add_to_calendar: 添加到日历
+    event_details: 事件详情
+    date: 日期
+    location: 地点
+    speakers: 发言人
+    host: 主办方 # TODO: host?
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:199]:
```suggestion
    host: 主办方
```

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 10 above)
Comment (jean-gui) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:160]: You can find an example of this block on https://www-staging.w3.org/blog/2020/blog-post/, look for "1 February 2021". It's used to display additional information about the blog post, usually update information. `post.notes_heading` is only used with screen readers

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 17 above)
Comment (jean-gui) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:62]: This class is explained at https://design-system.w3.org/styles/how-to-hide-and-show-things.html#visually-hiding-items. It visually hides content while keeping it accessible to screen readers. So in this case, a screen reader would say "Next page".

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 10 above)
Comment (xueyuanjia) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:160]: Hi @jennyliang220 , I think "注释说明" is the correct translation here.

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 12 above)
Comment (xueyuanjia) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:17]: @jennyliang220 How about using "相关内容" for "related content"?

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml (hunk 8 above)
Comment (xueyuanjia) [reply translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:140]: +1 to use a similar one for all languages. @jennyliang220 Let's use "来自其他组织的评价” which is more generic. :)

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 23]
```diff
@@ -1,205 +1,256 @@
...
     errors:
-        default_title: __notes.errors.default_title
+        default_title: 出现了一些问题
 events:
     # same_day = 1 when start_date == end_date
     date_range: >-
         {same_day, select,
             1  {{start_date}, {start_time} - {end_time} <b>{timezone}</b>}
-            other {{start_date}, {start_time} - {end_date}, {end_time} <b>{timezone}</b>}
+            其它 {{start_date}, {start_time} - {end_date}, {end_time} <b>{timezone}</b>}
```
Comment (jean-gui) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:192]: "other" cannot be translated, it's part of a condition:
 - if same_day is 1, use the first translation
 - if same_day is something else (other), use the second translation

File: translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml [hunk 24]
```diff
@@ -1,205 +1,256 @@
...
+        link_text: 访问{name}
     people:
-        tel: "__components.people.tel__Tel: <a href=\"tel:{number}\">{number}}</a>"
-        email: "__components.people.email__Email: <a href=\"mailto:{email}\">{email}</a>"
+        tel: "电话: <a href=\"tel:{number}\">{number}</a>"
+        email: "邮箱: <a href=\"mailto:{email}\">{email}</a>"
     evangelists:
-        list_title: __components.evangelists.list_title__Our {ecosystem_name} Evangelists
+        list_title: >-
+            Our {ecosystem_name} { count, plural,
```
Comment (jennyliang220) [inline translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml:125]:
```suggestion
            我们的<span lang="en">{ecosystem_name}</span> { count, plural,
```

**Review by xueyuanjia — APPROVED** [review]

**Pull request merged**